recipes/mark-nonconverted-reads/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: mark-nonconverted-reads
+  version: "1.0"
+
+source:
+  url: https://github.com/nebiolabs/mark-nonconverted-reads/archive/v1.0.tar.gz
+  sha256: 018e0cd58b55f07c5fac6b21dcff3379fc8f32b950c36f0be93bd4b696fe7d83
+
+requirements:
+  host:
+    - python >=3
+
+  run:
+    - python >=3
+    - pysam
+
+build:
+  noarch: python
```

Comment:
Can you add a build number (0)? That makes upgrades easier.

Reply:
Done!